packages/@webex/internal-plugin-dss/test/unit/spec/dss.ts

```diff
@@ -921,6 +1156,95 @@ describe('plugin-dss', () => {
         expect(Batcher.prototype.request.getCall(1).args).to.deep.equal(['id2']);
         expect(result).to.equal(response2);
       });
+
+      it('fails fails when mercury does not respond, later batches can still pass ok', async () => {
+        // Batch 1
+        const {
+          promises: [p1, p2, p3],
+        } = await testMakeBatchedRequests({
+          requests: [
+            {
+              id: 'req-id-1',
+              resource: '/lookup/orgid/userOrgId/identities',
+              bodyParams: {lookupValues: ['id1', 'id2', 'id3']},
+            },
+          ],
+          calls: [
+            {
+              method: 'lookup',
+              params: {id: 'id1', shouldBatch: true},
+            },
+            {
+              method: 'lookup',
+              params: {id: 'id2', shouldBatch: true},
+            },
+            {
+              method: 'lookup',
+              params: {id: 'id3', shouldBatch: true},
+            },
+          ],
+        });
+
+        // Batch 2
+        const {
+          promises: [p4],
+        } = await testMakeBatchedRequests({
+          requests: [
+            {
+              id: 'randomid',
+              resource: '/lookup/orgid/userOrgId/identities',
+              bodyParams: {lookupValues: ['id4']},
+            },
+          ],
+          calls: [
+            {
+              method: 'lookup',
+              params: {id: 'id4', shouldBatch: true},
+            },
+          ],
+        });
+
+        // Batch 1 - only 1 response out of 3
```

Comment:
"// Batch 1 - only 1 mercury response out of 2 received"
(I would say 2 responses out of 3, but it doesn't really count as a response until we have the full sequence)